src/codeu/chat/client/simplegui/CreateAccountPanel.java

```diff
@@ -1,42 +1,47 @@
 package codeu.chat.client.simplegui;
 
 import javax.swing.*;
+
+import codeu.chat.client.simplegui.NewAccount;
+
 import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
 
 /**
- * Created by Suveena on 5/18/17.
+ * @author  Suveena
+ * @date    5/18/17
+ * @brief   This class contains a button for the user to create an account.
+ *          This panel contains from top to bottom; a message, a button.
  */
 
 @SuppressWarnings("serial")
-public class CreateAccountPanel extends JPanel{
+public class CreateAccountPanel extends JPanel {
 
-    public CreateAccountPanel() {
+    CreateAccountPanel() {
         super(new GridBagLayout());
         initialize();
     }
 
     private void initialize() {
 
-        // This panel contains from top to bottom; a message, a button.
 
         // Set layout within panel
         JPanel InnerLayout = new JPanel();
         InnerLayout.setLayout(new BoxLayout(InnerLayout, BoxLayout.Y_AXIS));
 
         JLabel userQuestionLabel = new JLabel("Create an Account");
         JButton createAccountButton = new JButton("Get Started!");
-
+        createAccountButton.addActionListener(new ActionListener(){
```

Comment:
can you add a comment for what this is supposed to do?

ie: "onclick: show panel to create a new account"

Also nit: space between () and {